protos/sigstore_rekor.proto

```diff
@@ -101,10 +101,14 @@ message TransparencyLogEntry {
         // verification.
         KindVersion kind_version = 3 [(google.api.field_behavior) = REQUIRED];
         // The UNIX timestamp from the log when the entry was persisted.
+        // A timestamp is only provided by Rekor v1 logs and will not be
+        // present for Rekor v2 log entries.
         // The integration time MUST NOT be trusted if inclusion_promise
         // is omitted.
-        int64 integrated_time = 4 [(google.api.field_behavior) = REQUIRED];
+        int64 integrated_time = 4;
         // The inclusion promise/signed entry timestamp from the log.
+        // The SET may be provided by Rekor v1 logs, but log entries
```

Comment:
nit, SET isn't defined til later, can we say `inclusion_promise`?

Reply:
yeah